app/domain/errors.rb

```diff
@@ -584,7 +584,7 @@ module Constraints
       )
 
       NonPermittedRestrictionGiven = ::Util::TrackableErrorClass.new(
-        msg: "Role can't have one of these none permitted restrictions '{0-restrictions}'",
+        msg: "Role can't have standard claim or a mapped claim",
```

Comment:
This message a bit problematic for me.
It's thrown by infrastructural class [NonPermittedConstraint](https://github.com/cyberark/conjur/blob/0084b9d7f2a6b09eaa75093dd994f38ccee3e52d/app/domain/authentication/constraints/non_permitted_constraint.rb#L17) but contains the name of concrete features in it.
What will happen if any other authenticaficator will need to use the class?
What will happen if other parameters except mapped claims will affect permitted restrictions list?

Reply:
I prefer to keep the message as is as long it's an infrastructural message.
If we want to make things clear we should catch an exception and to have jet specific treatment for it.

Reply:
I think we should try to make it more understandable, the way it was it wasn't so clear